portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -0,0 +1,38 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
+/** Class containing the data for comments. */
+public final class Comments {
```

Comment:
Is this for single comment? If so, rename this class to Comment

Reply:
Ah makes sense, changed

Reply:
done.